src/static/css/nav-primary.less

```diff
@@ -0,0 +1,459 @@
+                                </ul>
+                            </section>
+                            <aside class="sub-nav_media">
+                                <a href="/the-bureau/">
+                                    <img src="../../static/img/nav-about-us-video.png"
+                                         alt="About Us video image">
+                                    <h4 class="short-desc u-mt15">
+                                        The CFPB: Working for you.
+                                    </h4>
+                                </a>
+                                <div class="short-desc">
+                                    This short video covers what the CFPB is and
+                                    how we are working for American consumers.
+                                </div>
+                            </aside>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+      notes:
+        -
+  tags:
+    - cfgov-nav-primary
+*/
+
+.primary-nav .list_link {
+    .u-link__colors(@black, @green);
+    .webfont-regular();
+}
+
+
+/* Sliding nav for portrait tablets and smaller
+   ========================================================================== */
+
+.respond-to-max(@tablet-max, {
+
+    html,
+    body {
+        height: 100%;
+    }
+
+    .u-show-on-desktop {
+        display: none;
+    }
+
+    // Remove extra padding added by .wrapper__match-content so the nav button
+    // is flush left with the screen.
+    .primary-nav_header-space.wrapper__match-content {
+        padding-left: 0;
+        padding-right: 0;
+    }
+
+    .sub-nav_menu-list {
+        width: 100%;
+
+    }
+
+    .sub-nav_menu-list .list_link {
+        // Override standard visited link color.
+        .u-link__colors(@pacific, @pacific, @pacific-50, @pacific, @navy);
+    }
+
+    .primary-nav_top-level-list.list-expanding {
+        // Nested list borders.
+        // Negative bottom margin to remove duplicate bottom border.
+        & .list-expanding {
+            margin: 0 0 -1px;
+        }
+
+        & .list_item {
+            display: block;
+            border-bottom: 1px solid @gray-50;
+            // Overriding .list-horizontal .list_item.
+            margin-right: 0;
+
+            &:first-child {
+                border-top: 1px solid @gray-50;
+            }
+        }
+
+        & .list_item__plain {
+            border-bottom: 0;
+            &:first-child {
+                border-top: 0;
+            }
+        }
+
+        & .list_item a {
+            display: block;
+            padding: (@grid_gutter-width / 2)
+                     @grid_gutter-width
+                     (@grid_gutter-width / 2)
+                     (@grid_gutter-width / 2);
+            position: relative;
+            background: @gray-20;
+            font-size: unit(18px / @base-font-size-px, em);
+        }
+
+        & > .list_item > a:hover {
```

Comment:
These link states could all be nested into the link block above for better code org.